### DISCUSSION TITLE
Discord / Matrix channels/rooms cleanup

### DISCUSSION THREAD

**Discussion by francesco-ooops** [OCA/odoo-community.org#104] [Ideas]
Following from #102 : some rooms are just full of spam messages and others haven't been messaged in a long time, so to clean up a bit we would remove any channel (in Discord) and related room (in Matrix) without contributions in the last year. 

If you have suggestions for a policy on how to create/maintain rooms on both platforms, we're open to suggestions.

Francesco

**Comment by yajo** [comment]
I just wanted to announce that the work is done. Special thanks to @francesco-ooops who did most of it!

**Comment by theangryangel** [comment]
Thank you for the effort Francesco <3 (and Jairo)

**Comment by vdewulf** [comment]
Thanks a lot @francesco-ooops!

**Comment by HekkiMelody** [comment]
Thank you both! :heart:

**Comment by hartscott** [comment]
Are there any new Discord servers active? All I've seen have been removed. Don't know why people have to ruin things... 🤦‍♂️ Wonder if OCA could make a server that we had to donate like $1 or $5 or something to join? Then it might (fingers crossed) keep the horrible people from destroying it. It's a beautiful project and a beautiful community, so would love to see a place that could be easily discussed in real(ish) time like Discord. Thanks for everything you guys do!

**Reply by theangryangel** [reply]
I’m not sure what servers you’ve joining, to my knowledge there is only https://discord.gg/Y2tdx9DP?

**Reply by hartscott** [reply]
OH! Kept googling and googling but couldn't locate the actual Discord server. Even went into Browse in Discord. Guess I was being silly. Thanks!

**Reply by francesco-ooops** [reply]
Hi, this link will be valid for 7 days: https://discord.gg/GEa5HfMf
